net/errors.ts

```diff
@@ -0,0 +1,2 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+export class SocketClosedError extends Error {}
```

Comment:
Hrm... Not sure if we should start a new file for this already. Just move this to`ws.ts` for now.

Reply:
done.